docs/reference/service-discovery.md

```diff
@@ -0,0 +1,145 @@
+# Discovery Service
+
+Service Discovery Service is used for the communication between multiple ASAB microservices in a server cluster.
+
+In order to work, the following requirements must be fulfilled:
+
+- Zookeeper connection and configuration must be the same for all services in the cluster.
+- Zookeeper container must be initialized in the service.
+- `asab.WebService` and `asab.WebContainer` must be initialized.
+- `asab.APIService` must be initialized.
+- Environment variables `NODE_ID`, `SERVICE_ID` and `INSTANCE_ID` must be set.
+
+
+## Advertising services into Zookeeper
+
+### API Configuration
+
+In order to propagate information about a microservice into Zookeeper, you need to include configuration for the web and Zookeeper containers.
```

Comment:
yes, but it might be default. 
It would be great to explain it further, so it is clear how each piece of information is propagated

Reply:
e.g. port, or instance id